logdevice/common/settings/SettingsUpdater.cpp

```diff
@@ -106,6 +106,14 @@ std::string SettingsUpdater::markdownDoc(bool include_deprecated) const {
 
   using std::endl;
 
+  // Markdown header
```

Comment:
We shouldn't mix Docusaurus specific code into the logdeviced code. Can't we concatenate the header from cmake/automation instead?